src/System.Data.Common/src/System.Data.Common.csproj

```diff
@@ -310,9 +313,14 @@
     <Reference Include="System.Threading" />
     <Reference Include="System.Threading.Thread" />
     <Reference Include="System.Transactions.Local" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' != 'uapaot' and '$(TargetGroup)' != 'netfx'">
     <Reference Include="System.Xml.ReaderWriter" />
     <Reference Include="System.Xml.XmlSerializer" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'uapaot'">
+    <ProjectReference Include="..\..\System.Private.Xml\src\System.Private.Xml.csproj" />
```

Comment:
This will get you into a weird state, which might still work, but essentially here you are building against a mix of contracts and implementations. I would instead of doing this, either building everything against contracts, or everything against implementations. That means that the ItemGroup above that has all of the `<Reference Include` should also have a condition of `'$(TargetGroup)' != 'uapaot'` and you should build against System.Private.Corelib and src\*csprojs.